lightning/src/ln/channel.rs

```diff
@@ -919,6 +923,39 @@ impl<'a, SP: Deref> ChannelPhase<SP> where
 	}
 }
 
+pub(super) struct UnconfirmedChannelContext {
+	/// A counter tracking how many ticks have elapsed since this unaccepted channel was
+	/// created. If this unaccepted channel reaches peer has yet to respond after reaching
+	/// `UNACCEPTED_CHANNEL_AGE_LIMIT_TICKS`, it will be force-closed and purged from memory.
+	///
+	/// This is so that we don't keep outbound request around which have not been accepted
```

Comment:
s/which/that

Reply:
I have cleaned up this section in [5a4a8bb](https://github.com/lightningdevkit/rust-lightning/pull/2725/commits/5a4a8bb45fa02c39fb9b54ed536e3dc879139348)
Thanks!